packages/account/src/Components/poi/status/unsupported/unsupported.tsx

```diff
@@ -49,25 +53,75 @@ const Unsupported = ({
     allow_poi_resubmission,
     handleViewComplete,
     onfido,
+    routeBackTo,
+    app_routing_history = [],
     ...props
-}: Partial<TUnsupported>) => {
+}: TUnsupported) => {
     const [detail, setDetail] = React.useState<number | null>(null);
+
     const toggleDetail = (index: number) => setDetail(index);
 
     const documents = getDocumentIndex({
         country_code,
     });
 
+    const from_platform = getPlatformRedirect(app_routing_history);
+
+    const onClickRedirectButton = () => {
+        const platform = platforms[from_platform?.ref as keyof typeof platforms];
+        const { is_hard_redirect = false, url = '' } = platform ?? {};
```

Comment:
why do we need to give this a default false value?

Reply:
not sure about that, It was implemented before.
I raised the question in our channel about this config, nobody has responded. SO I left previous implementation

Reply:
In case platform is undefined, we'll have the default values with correct types